src/namespaceActions.js

```diff
@@ -1,12 +1,22 @@
 import camelCase from './camelCase';
+import ownKeys from './ownKeys';
 import isPlainObject from 'lodash/isPlainObject';
+import includes from 'lodash/includes';
 
 const defaultNamespace = '/';
 
-function flattenActionMap(
-  actionMap,
+function hasGeneratorInterface(handler) {
```

Comment:
Maybe put this in a separate module, like `ownKeys`, `camelCase`, and `arrayToObject`?